schematic/store/synapse.py

```diff
@@ -512,7 +580,10 @@ def getFilesInStorageDataset(
 
                     if fullpath:
                         # append directory path to filename
-                        filename = (dirpath[0] + "/" + filename[0], filename[1])
+                        filename = (
+                            project_name + "/" + dirpath[0] + "/" + filename[0],
```

Comment:
I think this is not the expected behavior @GiaJordan @andrewelamb 
Please see ticket: https://sagebionetworks.jira.com/browse/FDS-2275

Reply:
I am thinking now that you might need to include project_name for the validation to work. Could you make file validation work without changing the Filename column?